grunt/helpers.js

```diff
@@ -196,11 +196,30 @@ module.exports = function(grunt) {
         var includes = grunt.config('includes');
         var excludes = grunt.config('excludes');
 
+        var isIncluded;
```

Comment:
Are both of these variables required? Wouldn't just having 'isIncluded' (true/false) suffice?

Reply:
Cheers @barry-learningpool I guess would be, I was just following on with the existing code trying to make as little impact as I could.

Would this be a recommended improvement then?

Reply:
Yeah that's fair enough @nilslp , would be an improvement